packages/react-icons/generateRollup.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+const path = require('path');
+// https://github.com/jprichardson/node-fs-extra/blob/master/docs
+const fs = require('fs-extra');
+// https://www.npmjs.com/package/glob
+const glob = require('glob');
+
+const outDir = path.join(__dirname, '/lib');
+// NOTE: esm comes first to increase chances of it being suggested first in auto-imports
+const outTypes = ['esm', 'cjs'];
+
+// // copy test output to fake react-icons package
```

Comment:
Is there long-term value in leaving these commented lines in?

Reply:
I'll be removing the test comments on line 12, as well as the commented out code block. I think the comment on line 9 is useful